<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arrayform</class>
 <widget class="QWidget" name="arrayform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Входные данные</string>
   </property>
   <widget class="QLabel" name="sizeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер массива   --&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sizeOfArrayLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewValuePushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="indexLineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="indexLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер элемента [] --&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sizeOfArrayPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Подтвердить</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="resetPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>881</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>591</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>Результаты</string>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Исходный массив</string>
    </property>
    <widget class="QListWidget" name="baseArrayListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>256</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>571</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Одержати новий масив, замінивши у вихідному масиві елементи, що стоять на парних місцях, їхніми квадратами.</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Одержати елементи масиву, які лежать у заданому діапазоні [n1, n2].</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="comboBoxPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>571</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>281</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Массив результатов</string>
   </property>
   <widget class="QListWidget" name="resultArrayListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>256</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
